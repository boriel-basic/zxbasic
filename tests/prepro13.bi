' Must return just PRINT "HELLO WORLD"


#define test x
#define x PRINT "HELLO WORLD"

test


