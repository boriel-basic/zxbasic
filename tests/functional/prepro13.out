#line 1 "prepro13.bi"






PRINT "HELLO WORLD"


